gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/ManagedHttpJsonChannel.java

```diff
@@ -47,23 +47,24 @@
 @BetaApi
 public class ManagedHttpJsonChannel implements HttpJsonChannel, BackgroundResource {
 
-  private static final ExecutorService DEFAULT_EXECUTOR =
-      InstantiatingExecutorProvider.newBuilder().build().getExecutor();
-
```

Comment:
Static DEFAULT_EXECUTOR error'd out on the unit tests. Same executor was being used and it was shutdown after the first unit test.